migrations/versions/970_add_brief_unsuccessful_and_closed_statuses.py

```diff
@@ -0,0 +1,29 @@
+"""Add 2 new status columns; one for 'unsuccessful', one for 'cancelled'.
+
+Revision ID: 970
+Revises: 960
+Create Date: 2017-08-25 11:38:57.947569
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
```

Comment:
Too many blank lines after imports.
PEP8 doesn't run here 😁

Reply:
Cool - I'll fix this in the migration PR (I have to update that anyway as someone else pushed a migration and the version numbers are out of date...)